tools/check/go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/pingcap/tidb-cdc/_tools
```

Comment:
why?

Reply:
it will rename as github.com/tikv/pd/_tools